resources/xsd/fastRTPS_profiles.xsd

```diff
@@ -1372,11 +1372,12 @@
 
     <!--| QoS POLICY TYPES DEFINITION |-->
     <!--QoS Data Sharing:
-        ├ kind         [string] ("AUTOMATIC", "ON", "OFF"),
-        ├ shared_dir   [string],
-        ├ domain_ids   [0~*],
-        |   └ domainID [uint32] (0~232)
-        └ max_domains  [uint32] -->
+        ├ kind                         [string] ("AUTOMATIC", "ON", "OFF"),
+        ├ shared_dir                   [string],
+        ├ domain_ids                   [0~*],
+        |   └ domainID                 [uint32] (0~232)
```

Comment:
Totally off-topic to the PR: Since this is not related with the DDS domain ID, the values here have no limits. They can be considered similar to the partitions.

Reply:
You're absolutely right

Reply:
59e3e626a